### PR TITLE
Update dependencies and fix issue 3

### PR DESCRIPTION
**Pull request by AlenPelin**

### CODE REVIEW

File: blog/next.config.js [hunk 1]
```diff
@@ -3,11 +3,21 @@ const withCustomBabelConfig = require("next-plugin-custom-babel-config");
 const withTypescript = require("@zeit/next-typescript");
 const withTranspileModules = require("next-plugin-transpile-modules");
 
-module.exports = withCustomBabelConfig(
+// https://github.com/wellcometrust/next-plugin-transpile-modules/issues/11
+function fixExternalsPlugin(config = {}) {
```
Comment (josephluck) [inline blog/next.config.js]: Hmm, seems as though this issue should be addressed up-stream in `next-plugin-transpile-modules`, but I'm happy to merge this in for now

File: blog/next.config.js [hunk 2]
```diff
@@ -3,11 +3,21 @@ const withCustomBabelConfig = require("next-plugin-custom-babel-config");
...
+  const { webpack } = config
+  config.webpack = (config, ...rest) => {
+    config.externals = config.externals || []
+    return webpack(config, ...rest)
+  }
+  return config
+}
+
+module.exports = fixExternalsPlugin(withCustomBabelConfig(
```
Comment (josephluck) [inline blog/next.config.js]: Can you nest the plugins with newlines as per the other plugins? Keeps them readable.

**Review by josephluck — CHANGES_REQUESTED** [review]

File: blog/next.config.js (hunk 1 above)
Comment (AlenPelin) [reply blog/next.config.js]: Added comments that it is workaround and it needs to be removed when module is fixed.

File: blog/next.config.js (hunk 2 above)
Comment (AlenPelin) [reply blog/next.config.js]: Refactored with use of next-compose-modules.

**Review by josephluck — APPROVED** [review]
LGTM, nice refactor

**Pull request merged**